ot/solvers.py

```diff
@@ -421,6 +423,12 @@ def solve(
                     max_iter = 1000
                 if tol is None:
                     tol = 1e-9
+                if grad == "last_step":
+                    if max_iter == 0:
```

Comment:
why this test? Also it need sto be tested

Reply:
a ok I see now still you need to test that the reaise happen if not called properly (new test)